crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -603,6 +606,47 @@ impl<'a> ExtensionLogic for ForgeExtension<'a> {
                 }
                 Ok(CheatcodeHandlingResult::Handled(output))
             }
+            "store" => {
+                let mut blockifier_state = BlockifierState::from(
+                    extended_runtime
+                        .extended_runtime
+                        .extended_runtime
+                        .extended_runtime
+                        .hint_handler
+                        .state,
+                );
+                let target = ContractAddress::from_(reader.read_felt());
+                let storage_address = reader.read_felt();
+                store(
+                    &mut blockifier_state,
+                    target,
+                    &storage_address,
+                    reader.read_felt(),
+                )
+                .expect("Failed to store");
+                Ok(CheatcodeHandlingResult::Handled(vec![]))
+            }
+            "load" => {
+                let mut blockifier_state = BlockifierState::from(
+                    extended_runtime
+                        .extended_runtime
+                        .extended_runtime
+                        .extended_runtime
+                        .hint_handler
+                        .state,
+                );
+                let target = ContractAddress::from_(reader.read_felt());
+                let storage_address = &reader.read_felt();
+                let loaded =
+                    load(&mut blockifier_state, target, storage_address).expect("Failed to load"); // Should not happen
```

Comment:
```suggestion
                    load(&mut blockifier_state, target, storage_address).expect("Failed to load");
```
probably unnecessary